backend/src/main/java/com/alex/eshop/dto/itemDTOs/ItemDTO.java

```diff
@@ -7,6 +7,7 @@ public class ItemDTO {
     private String categoryName;
     private String description;
     private String imageSrc;
+    private Boolean hasImage;
```

Comment:
`Boolean hasImage;` can be removed, as you can replace it with `imageSrc == null || imageSrc.isEmpty()` or using util method `ObjectUtils.isEmpty()` or `StringUtils.isBlank()`